static/_redirects

```diff
@@ -550,7 +550,7 @@ https://kubernetes-io-v1-7.netlify.com/*    https://v1-7.docs.kubernetes.io/:spl
 /docs/setup/on-premises-vm/dcos/     /docs/setup/production-environment/on-premises-vm/dcos/   301
 /docs/setup/on-premises-vm/ovirt/     /docs/setup/production-environment/on-premises-vm/ovirt/   301
 /docs/setup/windows/intro-windows-in-kubernetes/     /docs/setup/production-environment/windows/intro-windows-in-kubernetes/   301
-/docs/setup/windows/user-guide-windows-nodes/     /docs/setup/production-environment/windows/user-guide-windows-nodes/   301
+/docs/setup/windows/user-guide-windows-nodes/     /docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/   301
```

Comment:
Add a redirect from /docs/setup/production-environment/windows/user-guide-windows-nodes/ to /docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/ as well.